src/patternfly/components/Label/docs/code.md

```diff
@@ -0,0 +1,10 @@
+## Overview
+
+The label component is to be used in a table. When the table component has a compact modifier on it, the label is required to have one too in order to adjust the font size.
```

Comment:
I'm not sure the label is only for use in the table. In the design it's shown in a form as well. I think you could just be more generic here with the description.

Reply:
The new sketch file shows it not in the form, I dont think it will be in forms anymore..but double checking with Kyle.

Reply:
ok cool, ya go with whatever Kyle says.